scripts/apps/authoring/authoring/directives/AuthoringDirective.js

```diff
@@ -420,7 +420,7 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
 
             $rootScope.publishAfterTansa = function () {
                 if (!onlyTansaProof) {
-                    $scope.saveTopbar().then(continueAfterPublish ? $scope.publishAndContinue : $scope.publish);
+                    $scope.saveTopbar().then(continueAfterPublish === true ? $scope.publishAndContinue : $scope.publish);
```

Comment:
This is because of `$scope.publish = function(continueOnPublish) {`.

Here I pass true: 
```$scope.publishAndContinue = function() {
                $scope.publish(true).then(function(published) {```

And on other occasions, on publish - item is passed, so there is object, so it is needed to check only for true.